dev/release/src/release.ts

```diff
@@ -518,7 +518,7 @@ cc @${config.captainGitHubUsername}
                                     const fullPath = path.join(updateDirectory, file)
                                     let updateContents = readFileSync(fullPath).toString()
                                     if (notPatchRelease) {
-                                        const releaseHeader = `## ${previousVersion} -> ${nextVersion}`
+                                        const releaseHeader = `## v${previousVersion} -> v${nextVersion}`
```

Comment:
Could we use a Unicode arrow here for more beauty? 
```suggestion
                                        const releaseHeader = `## v${previousVersion} ➔ v${nextVersion}`
```

https://unicode-table.com/en/2794/

Reply:
Updated existing docs as well.